src/screens/TransactionSuccess.tsx

```diff
@@ -0,0 +1,55 @@
+import CheckCircleIcon from "@assets/icons/check-circle.svg";
+import { ReactNode } from "react";
+import { View, StyleSheet } from "react-native";
+
+import Button from "@/components/buttons/Button";
+import Heading from "@/components/typography/Heading";
+import { spacing } from "@/constants/DesignTokens";
+
+type ConfirmProps = {
```

Comment:
It should follow the pattern of name and `Props`

```suggestion
type TransactionSuccessProps = {
```